packages/cli/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# CHANGELOG.md
+
+- 4.0.x major overhaul to package structure and functionality.
```

Comment:
IMHO changelog should be done by squashing the commits on PR's and reference them on top level. 
So maybe we can have it in root folder to be seen. I generataly tend to have it on website as well.

Reply:
Agree--for now this PR was an attempt to get npm's entry fixed. Would want to leave the overall structure to the maintainers.

Reply:
I don't think we need any extra CHANGELOG because we already have `Releases` section of this repo.